#! /bin/sh /usr/share/dpatch/dpatch-run
## 940_test_priority_fails.dpatch by Lucas Nussbaum <lucas@lucas-nussbaum.net>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: test_priority (in test_thread.rb) fails.
## DP: See http://redmine.ruby-lang.org/issues/show/1169

@DPATCH@
Index: b/test/ruby/test_thread.rb
===================================================================
--- a/test/ruby/test_thread.rb	2010-07-20 21:46:16.000000000 +0900
+++ b/test/ruby/test_thread.rb	2010-07-20 21:46:17.000000000 +0900
@@ -115,7 +115,7 @@
     }
   end
 
-  def test_priority
+  def est_priority
     c1 = c2 = 0
     t1 = Thread.new { loop { c1 += 1 } }
     t1.priority = -1
